core/tests/test_apps.nem.transfer.py

```diff
@@ -102,19 +102,29 @@ def test_create_transfer_with_mosaic(self):
 
 def _create_msg(network: int, timestamp: int, fee: int, deadline: int,
                 recipient: str, amount: int, mosaics: int = 0):
-    m = NEMSignTx()
-    m.transaction = NEMTransactionCommon()
-    m.transaction.network = network
-    m.transaction.timestamp = timestamp
-    m.transaction.fee = fee
-    m.transaction.deadline = deadline
-    m.transfer = NEMTransfer()
-    m.transfer.recipient = recipient
-    m.transfer.amount = amount
-    m.transfer.mosaics = list()
-    for i in range(mosaics):
-        m.transfer.mosaics.append(NEMMosaic())
-    return m
+    transaction = NEMTransactionCommon(
+        network=network,
+        timestamp=timestamp,
+        fee=fee,
+        deadline=deadline,
+    )
+
+    mosaic_kwargs = {
+        "namespace": "abc",
+        "mosaic": "abc",
+        "quantity": 5,
+    }
+    transfer = NEMTransfer(
+        recipient=recipient,
+        amount=amount,
+        mosaics=[NEMMosaic(**mosaic_kwargs) for _ in range(mosaics)],
```

Comment:
why not list the kwargs here explicitly?

Reply:
I somehow thought I will be using it more than once. Changed in 329a038